front/components/assistant/conversation/actions/VisualizationActionIframe.tsx

```diff
@@ -197,7 +197,7 @@ export function VisualizationActionIframe({
                     height: !isErrored ? `${contentHeight}px` : "100%",
                     minHeight: !isErrored ? "96" : undefined,
                   }}
-                  className={classNames("max-h-[60vh] w-full")}
+                  className={classNames("w-full")}
```

Comment:
We may want to keep a max height, maybe keeping it in sync with a value we also tell the model in the prompt.
A 3000px high viz (or anything that takes the entire screen really) won't be a very good experience IMO

Reply:
changed this to a max height of 600px that we also give to the prompt

Reply:
does it make sense to use pixel values or is responsiveness becoming an issue here?

Reply:
not sure,  I think if it tends to not look too distorted on all sorts of computer screen (from 13 inch laptop screen to big 4k monitor) pixel height may be fine. 

a portrait plot on a phone screen is always going look bad (if we really wanted to support those we'd probably need to have a button and display it in landscape mode, but using viz on phone is likely super niche for now so I wouldn't bother yet)

Reply:
Looks ok on the 2 screens i got here

Reply:
also may cause an issue when using a browser not in full screen, but I feel like it makes sense that the user should use a relatively big window to visualize a chart